pkg/util/detector/detector.go

```diff
@@ -955,8 +954,6 @@ func (d *ResourceDetector) ReconcileClusterResourceBinding(key util.QueueKey) er
 
 	klog.Infof("Reconciling cluster resource binding(%s)", binding.Name)
 	switch binding.Spec.Resource.Kind {
-	case util.DeploymentKind:
-		return d.AggregateDeploymentStatus(binding.Spec.Resource, binding.Status.AggregatedStatus)
```

Comment:
Why?

Reply:
ClusterResourceBinding should be cluster-scope resource

Reply:
Nice finding!

Reply:
Wow, right, since `ClusterResourceBinding` can only `binding` a cluster-scoped resource.

Reply:
But, with this patch, the `ReconcileClusterResourceBinding` actually does nothing.  right?

Reply:
yes, but it could confuse others.

Reply:
So, can you help remove this and we can add them back if there is a need in the future?

Reply:
You mean keep the code `case util.DeploymentKind`? I perfect change to `clusterrole` as a demo.

Reply:
> You mean keep the code case util.DeploymentKind?

No, I mean remove the whole method. All dead codes should not exist.